tfjs-backend-webgpu/src/index.ts

```diff
@@ -15,14 +15,19 @@
  * =============================================================================
  */
 
+import './flags_webgpu';
 import * as tf from '@tensorflow/tfjs-core';
 import * as Shaderc from '@webgpu/shaderc';
 
 import {WebGPUBackend} from './backend_webgpu';
 
 tf.registerBackend('webgpu', async () => {
   const shaderc = await Shaderc.instantiate();
-  const adapter = await navigator.gpu.requestAdapter({});
+  let gpuDescriptor: GPURequestAdapterOptions;
+  if (tf.env().get('WEBGPU_USE_LOW_POWER_GPU')) {
+    gpuDescriptor = {powerPreference: 'low-power'};
+  }
```

Comment:
Actually, you might want to specifically request `'high-performance'` in the other case.

Right now, high-performance is the default in Chrome, but you should not rely on this.

```typescript
const gpuDescriptor: GPURequestAdapterOptions = {
  powerPreference: tf.env().get('WEBGPU_USE_LOW_POWER_GPU') ? 'low-power' : 'high-performance',
};
```

Or maybe you want to swap it around:

```typescript
const gpuDescriptor: GPURequestAdapterOptions = {
  powerPreference: tf.env().get('WEBGPU_USE_HIGH_PERFORMANCE_GPU') ? 'high-performance' : 'low-power',
};
```

Or maybe you want to also allow default selection:

```typescript
let gpuDescriptor: GPURequestAdapterOptions | undefined;
switch (tf.env().get('WEBGPU_POWER_PREFERENCE')) {
  case 'high-performance':
    gpuDescriptor = { powerPreference: 'high-performance' };
    break;
  case 'low-power':
    gpuDescriptor = { powerPreference: 'low-power' };
    break;
  // defaults to no powerPreference
};
```

Reply:
Done. Thanks for your suggestion.